scripts/unit/github-actions/create-pull-request-spec.js

```diff
@@ -0,0 +1,86 @@
+const { expect } = require('chai')
```

Comment:
Random q, I wonder if a better way to test if this is working would be to just have a local check in the monorepo. Eg, when you run `yarn install`, we would (somehow) check the last time the snapshot metadata was updated. If it's been more than <time period>, we could either error or warn "snapshot hasn't been updated in a week; is the automation working?"

Just an idea - if this GHA stopped working, or we changed to use another VCS service, we'd need to remember to do this. Not a requirement for this PR, specifically - just floating an idea to make the monorepo more self maintaining.